bgpd/bgp_labelpool.c

```diff
@@ -517,7 +624,7 @@ void bgp_lp_event_chunk(uint8_t keep, uint32_t first, uint32_t last)
 finishedrequest:
 		lp_fifo_del(&lp->requests, lf);
 		XFREE(MTYPE_BGP_LABEL_FIFO, lf);
-	}
+}
```

Comment:
this seems wrong

Reply:
This = = indent? Oops.

Reply:
Hmm. It seems this indentation is introduced by 'git clang-format'

Reply:
Or, it could be that the label + following statements should be outside the loop (slaps forehead)